### PR TITLE
Get region

### PR DESCRIPTION
**Pull request by david-deboer**
This adds code to return the "region" of an antenna or antenna list.  The regions are [N, E, W] for the core antennas and [A, B] for the outriggers (A is the inner ring, B the outer).  Query inputs can be 123, '123', 'HH123', 'A123', etc. If given a list it returns a list, else it returns a string.  If given a part name ('HA325') for an outrigger antenna it will check the prefix and issue a warning if not in agreement.

### CODE REVIEW

File: hera_mc/geo_sysdef.py [hunk 1]
```diff
@@ -43,6 +44,63 @@
...
+        A ring (inner)
+        B ring (outer)
+
+    Parameter
+    ---------
+    ants : str, int or list
+            Antenna number or HERA part number of antenna.
+    """
+    return_as = 'list'
+    if not isinstance(ants, list):
```
Comment (jsdillon) [inline hera_mc/geo_sysdef.py:95]: You might want to also allow the input to be tuples or numpy arrays.

File: hera_mc/tests/test_geo_location.py [hunk 2]
```diff
@@ -39,6 +39,15 @@ def test_geo_sysdef():
     assert int(a['HH123']['E']) == 540945
 
 
+def test_geo_regions():
```
Comment (jsdillon) [inline hera_mc/tests/test_geo_location.py:42]: It'd be good to see more tests. I'd try having lists with multiple entries. I'd try all the different types you mentioned in the PR, e.g. `123, '123', 'HH123', 'A123'`

**Review by jsdillon — CHANGES_REQUESTED** [review]
Just a few suggestions but basically fine.

File: hera_mc/geo_sysdef.py (hunk 1 above)
Comment (david-deboer) [reply hera_mc/geo_sysdef.py]: I made it more flexible to allow other list-y options, using TypeError vs ValueError.  Did similar in the _get_region module.

File: hera_mc/tests/test_geo_location.py (hunk 2 above)
Comment (david-deboer) [reply hera_mc/tests/test_geo_location.py:42]: Added

**Review by jsdillon — APPROVED** [review]

**Pull request merged**